03-intro-to-python.Rmd

```diff
@@ -1,9 +1,298 @@
 
 # Intro to Python
 
+Python is a popular programming language that was created by Guido van Rossum and released in 1991. 
+
+Python is supported by multiple libraries that support data science tasks:
+
+- NumPy for numerical computing with multidimensional arrays
+- pandas for data manipulation and analysis with data frames
+- Matplotlib for data visualization
+
+## Main Differences between R and Python
+
+The main difference between Python and R is that Python is a general-purpose programming language, while R is a statistical programming language. This means that Python is good at multiple things and can do most things, whereas R is very good at statistical analysis, but not as good at other things as Python.
+
+When you want to document your code, you can use Jupyter Notebooks. Jupyter Notebooks are an open source web application for easily sharing documents that contain your live Python code, equations, visualizations and data science explanations. 
```

Comment:
You can document non-notebook code, but notebooks are just useful for making easy to read documents with the code, documentation and output all in one place.